docs/docs/examples/vector_stores/AzureAISearchIndexDemo.ipynb

```diff
@@ -118,18 +117,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
     "search_service_api_key = \"YOUR-AZURE-SEARCH-SERVICE-ADMIN-KEY\"\n",
     "search_service_endpoint = \"YOUR-AZURE-SEARCH-SERVICE-ENDPOINT\"\n",
-    "search_service_api_version = \"2023-11-01\"\n",
+    "search_service_api_version = \"2024-07-01\"\n",
     "credential = AzureKeyCredential(search_service_api_key)\n",
     "\n",
     "\n",
     "# Index name to use\n",
-    "index_name = \"llamaindex-vector-demo\"\n",
+    "index_name = \"llamaindex-vector-demo-compression\"\n",
```

Comment:
Is it a good idea for the only example we have in the notebook to be one with compression? Does this use oversampling/rescoring as well? How about semantic reranking? Would be good to have an example that doesn't only focus on compression but that shows strong balance between quality and compression.

Reply:
Fair, I'm thinking we leave the default notebook uncompressed but just show a comment on how in 1-line you can enable SQ/BQ w/reranking/oversampling. 

Generally, LlamaIndex notebook samples are a more of a basic quickstart. I'll leave the Hybrid Semantic Reranking query example though.